Also, this causes two test failures in `test_heap.py`:

```
=============================================== FAILURES ===============================================
______________________________________ TestBinaryHeap.test_insert ______________________________________

self = <test_heap.TestBinaryHeap testMethod=test_insert>

    def test_insert(self):
        # Before insert 2: [0, 4, 50, 7, 55, 90, 87]
        # After insert:  [0, 2, 50, 4, 55, 90, 87, 7]
        self.min_heap.insert(2)
        self.assertEqual([0, 2, 50, 4, 55, 90, 87, 7],
                         self.min_heap.heap)
>       self.assertEqual(7, self.min_heap.currentSize)
E       AttributeError: 'BinaryHeap' object has no attribute 'currentSize'

tests/test_heap.py:31: AttributeError
____________________________________ TestBinaryHeap.test_remove_min ____________________________________

self = <test_heap.TestBinaryHeap testMethod=test_remove_min>

    def test_remove_min(self):
        ret = self.min_heap.remove_min()
        # Before remove_min : [0, 4, 50, 7, 55, 90, 87]
        # After remove_min: [7, 50, 87, 55, 90]
        # Test return value
        self.assertEqual(4, ret)
        self.assertEqual([0, 7, 50, 87, 55, 90],
                         self.min_heap.heap)
>       self.assertEqual(5, self.min_heap.currentSize)
E       AttributeError: 'BinaryHeap' object has no attribute 'currentSize'

tests/test_heap.py:41: AttributeError
======================================= short test summary info ========================================
FAILED tests/test_heap.py::TestBinaryHeap::test_insert - AttributeError: 'BinaryHeap' object has no a...
FAILED tests/test_heap.py::TestBinaryHeap::test_remove_min - AttributeError: 'BinaryHeap' object has ...
==================================== 2 failed, 392 passed in 6.80s =====================================
```

--- a/algorithms/heap/binary_heap.py
+++ b/algorithms/heap/binary_heap.py
@@ -73,7 +73,8 @@ class BinaryHeap(AbstractHeap):
                 self.heap[i], self.heap[i//2] = self.heap[i//2], self.heap[i]
             i = i // 2
 
-    """
+    def insert(self, val):
+         """
         Method insert always start by inserting the element at the bottom.
         It inserts rightmost spot so as to maintain the complete tree property.
         Then, it fixes the tree by swapping the new element with its parent,
@@ -81,9 +82,6 @@ class BinaryHeap(AbstractHeap):
         perc_up the minimum element
         Complexity: O(logN)
     """
-
-    def insert(self, val):
-        """Inserts the given value to the heap"""
         self.heap.append(val)
         self.current_size = self.current_size + 1
         self.perc_up(self.current_size)

--- a/algorithms/heap/binary_heap.py
+++ b/algorithms/heap/binary_heap.py
@@ -74,21 +74,21 @@ class BinaryHeap(AbstractHeap):
             i = i // 2
 
     def insert(self, val):
-         """
+        """
         Method insert always start by inserting the element at the bottom.
         It inserts rightmost spot so as to maintain the complete tree property.
         Then, it fixes the tree by swapping the new element with its parent,
         until it finds an appropriate spot for the element. It essentially
         perc_up the minimum element
         Complexity: O(logN)
-    """
+        """
         self.heap.append(val)
         self.current_size = self.current_size + 1
         self.perc_up(self.current_size)
 
-    """
+        """
         Method min_child returns the index of smaller of 2 children of parent at index i
-    """
+        """
 
     def min_child(self, i):
         if 2 * i + 1 > self.current_size:  # No right child

--- a/tests/test_heap.py
+++ b/tests/test_heap.py
@@ -28,7 +28,7 @@ class TestBinaryHeap(unittest.TestCase):
         self.min_heap.insert(2)
         self.assertEqual([0, 2, 50, 4, 55, 90, 87, 7],
                          self.min_heap.heap)
-        self.assertEqual(7, self.min_heap.currentSize)
+        self.assertEqual(7, self.min_heap.current_size)
 
     def test_remove_min(self):
         ret = self.min_heap.remove_min()
@@ -38,7 +38,7 @@ class TestBinaryHeap(unittest.TestCase):
         self.assertEqual(4, ret)
         self.assertEqual([0, 7, 50, 87, 55, 90],
                          self.min_heap.heap)
-        self.assertEqual(5, self.min_heap.currentSize)
+        self.assertEqual(5, self.min_heap.current_size)
 
 
 class TestSuite(unittest.TestCase):

